docker-image/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.7
+
```

Comment:
Try to get familiar with `hadolint` and apply it to your `Dockerfile` to see what suggestions it gives you in terms of improving the code, here's a [post](https://www.howtogeek.com/devops/how-to-use-hadolint-to-lint-your-dockerfiles/) to learn more about it

Reply:
I've used hadolint to format the Dockerfile accordingly :)